Ordered sets module java (#247)
cpinitiative/usaco-guide: content/3_Silver/Intro_Ordered.mdx

<JavaSection>

The ordered set (TreeSet) allows for a multitude of additional operations: 
`first`: returns the lowest element in the set
`last()`: returns the greatest element in the set
`lower(E v)`: returns the greatest element strictly less than `v`
`floor(E v)`: returns the greatest element less than or equal to `v`
`higher(E v)`: returns the least element strictly greater than `v`

> `ceiling(E v)`: returns the least element greater than or equal to `v`

Reviewer: Why isn't this formatted as a list?

}
```

</CPPSection>

<JavaSection>

<Warning>

> The java version is too slow for the cses grading system, for a passing implementation use c++ instead.

Reviewer: ```suggestion
Java solutions are too slow for the CSES. Use C++ instead to get AC.
```
